### DISCUSSION TITLE
How to Pull Request for Onboarding

### DISCUSSION THREAD

**Discussion by techandy42** [castorini/pyserini#1701] [Q&A]
Hello, my name is Andy Lee, a current CS student at U of Waterloo.

I finished all the onboarding tasks for Anserini and Pyserini.

How should I pull requests for the reproduction logs? Should I fork the repos, add my changes, and sync the repos, or add my changes to the cloned repos and push my changes?

I currently worked on locally cloned repos.

**Comment by lintool** [comment]
Yes, look at recent PRs to see examples of other people doing the same.
